core/templates/pages/about-page/about-page.module.ts

```diff
@@ -19,6 +19,9 @@
 import 'core-js/es7/reflect';
 import 'zone.js';
 
+import 'third-party-imports/angular-js.import';
+import 'third-party-imports/headroom.import';
+
 angular.module('oppia', [
   'dndLists', 'headroom', 'infinite-scroll', 'ngAnimate',
   'ngAudio', require('angular-cookies'), 'ngJoyRide', 'ngMaterial',
```

Comment:
Why you are at it could you please try to limit the number of libs that we import here? I think for the basic pages you can inspire in pending deletion page or delete account page.

Reply:
Ditto below

Reply:
I'll do it in my further PRs.

Reply:
Okay.